int32 jointPos # position of the controlled joint
int32 actuatorPos # position of the actuator in encoder counts
int32 actuatorVel # velocity of the actuator in counts/s
uint16 actuatorCurrent # current drawn by the actuator
int16 elasticDisplacement # displacement of the SE element in encoder counts
int16 contractileDisplacement # displacement of the CE element

